tracer/src/Datadog.Trace/Logging/LogsInjection/LibLogScopeEventSubscriber.cs

```diff
@@ -271,12 +258,6 @@ private static void InitResolvers()
                     CustomSerilogLogProvider.IsLoggerAvailable,
                     () => new CustomSerilogLogProvider()));
 
-            LogProvider.LogProviderResolvers.Insert(
-                1,
-                Tuple.Create<LogProvider.IsLoggerAvailable, LogProvider.CreateLogProvider>(
-                    NoOpSerilogLogProvider.IsLoggerAvailable,
-                    () => new NoOpSerilogLogProvider()));
-
```

Comment:
I think this will change the current behaviour for customers that have 2 log frameworks available (a scenario we've seen in practice)? If we decide this _is_ safe, we need to update the insertion indices for providers below

Reply:
Ah I didn't think of that 🤦🏼 Essentially what I wanted to avoid was a situation where we use a NoOp log provider but we still do a lot of calls to `OpenMappedContext` because the amount of things that could go wrong is high. Perhaps we keep the log provider's there unchanged, but if we resolve to a NoOp log provider we don't subscribe to any span events?